### ISSUE TITLE
Fix to `.deb` for Debian install

### ISSUE THREAD

**Issue by ahttraga** [atlas-engineer/nyxt#1222]
**Describe the bug**
Under debian unstable the Ubuntu package for version 2-pre-release-5 does not install because the dependency package `enchant` is called `enchant-2` in Debian.

**Precise recipe to reproduce the issue**
`sudo apt install ./nyxt_2_amd64.deb`

**Fix**
Repackage `.deb` for Debian, replacing `enchant` dependency with `enchant-2`:
```bash
ar x nyxt_2_amd64.deb
tar xzf control.tar.gz
sed -i 's/enchant/enchant-2/g' control
tar czf control.tar.gz md5sums control
ar rcs nyxt_2_debian_amd64.deb debian-binary control.tar.gz data.tar.gz
```
Then `apt install` of the new Debian-compatible package `nyxt_2_debian_amd64.deb` does not fail.

**Information**
- OS name+version: Linux *** 5.10.0-2-amd64 #1 SMP Debian 5.10.9-1 (2021-01-20) x86_64 GNU/Linux
- Debian version: Debian GNU/Linux bullseye/sid
- Graphics card and driver:
- Desktop environment / Window manager name+version:
- How you installed Nyxt (Guix pack, package manager, build from source):
- Information from command copy-system-information:
Nyxt version: 2-pre-release-5 
Operating system kernel: Linux 5.10.0-2-amd64
Lisp implementation: SBCL 2.0.10
Features: (WEBKIT2 WEBKIT2-2.30 WEBKIT2-TRACKING WEBKIT2-MUTE WEBKIT2-EMOJI
           WEBKIT2-MEDIA WEBKIT2-SANDBOXING GTK-3-22 GTK-3-20 GTK-3-18 GTK-3-16
           GTK-3-14 GTK-3-12 GTK-3-10 GTK-3-8 GTK-3-6 GTK-3-4 GTK GDK-3-22
           GDK-3-20 GDK-3-18 GDK-3-16 GDK-3-14 GDK-3-12 GDK-3-10 GDK-3-8
           GDK-3-6 GDK-3-4 CAIRO-1-10 CAIRO-1-12 GDK-PIXBUF GLIB-2-30 GLIB-2-32
           GLIB-2-34 GLIB-2-36 GLIB-2-38 GLIB-2-40 GLIB-2-42 GLIB-2-44
           GLIB-2-46 GLIB-2-48 GLIB-2-50 GLIB-2-52 GLIB-2-54 GLIB-2-56 GLIB
           FSET-EXT-STRINGS SWANK PLUMP-UTF-32 GLOBAL-VARS DECLARE-TYPES
           PARENSCRIPT NAMED-READTABLES OSICAT-FD-STREAMS 21BIT-CHARS CHUNGA
           FLEXI-STREAMS CLOSER-MOP CL-PPCRE-UNICODE CL-UNICODE
           BORDEAUX-THREADS SPLIT-SEQUENCE GRAY-STREAMS FAST-IO-SV FAST-IO
           SBCL-USES-SB-ROTATE-BYTE ASDF-SYSTEM-CONNECTIONS CL-JSON-CLOS
           CL-JSON THREAD-SUPPORT CL-PPCRE FLAT-NAMESPACE X86-64 UNIX CFFI
           FLAT-NAMESPACE SEQUENCE-EMPTYP QUICKLISP ASDF3.3 ASDF3.2 ASDF3.1
           ASDF3 ASDF2 ASDF OS-UNIX NON-BASE-CHARS-EXIST-P ASDF-UNICODE X86-64
           GENCGC 64-BIT ANSI-CL COMMON-LISP ELF IEEE-FLOATING-POINT LINUX
           LITTLE-ENDIAN PACKAGE-LOCAL-NICKNAMES SB-CORE-COMPRESSION SB-LDB
           SB-PACKAGE-LOCKS SB-THREAD SB-UNICODE SBCL UNIX)

**Comment by Ambrevar** [comment]
Should be fixed with f050d7674d1c40bb74badd0903d988e92610e6b1.

I'll rebuild the .deb package soon.

**Comment by ahttraga** [comment]
Here is the [Debian bug report](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=947979) about the transition from `enchant` to `enchant-2`. The binary under Debian is `/usr/bin/enchant-2` instead of `/usr/bin/enchant`. I thought this may somehow break `nyxt` and `cl-enchant` but on my system `spell-check-word` works fine.

**Comment by Ambrevar** [comment]
We use cl-enchant which loads "libenchant", regardless of the version:

https://github.com/tlikonen/cl-enchant/blob/master/load-enchant.lisp

Actually we don't need to depend on `enchant-2` on Debian-based system,
we could just depend on `libenchant-2*`, but the problem is that the
versioning on Debian is a moving target that's hard to follow, so it's
just easier to depend on `enchant-2`.

List of files for libenchant:

https://packages.ubuntu.com/groovy/amd64/libenchant-2-2/filelist

Closing, feel free to reopen if you encounter another issue.
